static/js/components/tiles/top_event_tile.tsx

```diff
@@ -107,13 +125,20 @@ export function TopEventTile(props: TopEventTilePropType): JSX.Element {
                   return (
                     <tr key={i}>
                       <td className="rank">{i + 1}</td>
-                      <td>
-                        <a href={`/browser/${event.placeDcid}`}>
-                          {isUnnamedEvent(event.placeName)
-                            ? "Unnamed event"
-                            : event.placeName}
-                        </a>
-                      </td>
+                      {showNameColumn && (
+                        <td>
+                          <a href={`/browser/${event.placeDcid}`}>
+                            {getEventName(event)}
+                          </a>
+                        </td>
+                      )}
+                      {showPlaceColumn && (
+                        <td>
+                          {eventPlaces[event.placeDcid]
+                            ? eventPlaces[event.placeDcid].name
```

Comment:
please also add a link to the event here

Reply:
should we always link the event here or only when there's no event name?

Reply:
updated to add a link only when there's no event name